src/main/java/graphql/schema/transform/VisibleFieldPredicate.java

```diff
@@ -0,0 +1,25 @@
+package graphql.schema.transform;
+
+
+import graphql.schema.GraphQLFieldDefinition;
+import java.util.function.Predicate;
+
+/**
+ * Predicate used during a {@link FieldVisibilitySchemaTransformation} to test whether a field should be visible.
+ */
+public interface VisibleFieldPredicate extends Predicate<GraphQLFieldDefinition> {
```

Comment:
Please mark it as Private or Public matching the FieldVisibilitySchemaTransformation

Reply:
@PubicSPI

Reply:
Done